core/b2b/script-dev.tsx

```diff
@@ -4,16 +4,19 @@
 import Script from 'next/script';
 
 import { useB2BAuth } from './use-b2b-auth';
+import { useB2BCart } from './use-b2b-cart';
 
 interface DevProps {
   storeHash: string;
   channelId: string;
   hostname: string;
   token?: string;
+  cartId?: string | null;
```

Comment:
```suggestion
  cartId?: string;
```

can we remove `null` from the equation? 

```tsx
cartId={session?.user?.cartId ?? undefined}
```